### DISCUSSION TITLE
Search Result paging does not work within Teams (Viva Connections)

### DISCUSSION THREAD

**Discussion by mionline** [microsoft-search/pnp-modern-search#2781] [Ideas]
Opening a page with search results within Viva Connections (client or browser), the paging does not work - no reaction on clicking on the different pages (numbers). Within SharePoint it works fine.

**Comment by kasperbolarsen** [comment]
I have just checked and can confirm :-(

In SharePoint the link in the paginationContainer is https://[Tenant].sharepoint.com/#  and in Teams it is https://[Tenant].sharepoint.com/?app=portals#

**Comment by wobba** [comment]
Feel free to use the complete issue template when logging bugs. Which is why I converted this to a discussion. I'll tag this as an improvement idea and not a bug until someone prove it's the web parts, and not SPFx within Teams which is the issue.
